sympy/core/relational.py

```diff
@@ -1234,6 +1227,9 @@ def is_ge(lhs, rhs):
                 rv = diff.is_extended_nonnegative
                 if rv is not None:
                     return rv
+                rv = diff.evalf(2).is_extended_nonnegative
```

Comment:
Some expressions like `I**I` were returning `None` for `.is_extended_nonegative` check (Originally these were handled by the `_n2()` call). So to handle such cases I had to do a forced evaluation over here (done after the initial evaluation) to make sure that such expressions return something on a `is_ge()` call instead of just terminating at the above if condition. 

Will this forced evaluation over here ever affect the big number comparison in any case ?